src/common/components/moteinnkalling/MoteinnkallingContent.tsx

```diff
@@ -15,24 +16,36 @@ const texts = {
 
 interface Props {
   dialogmoteData: UseQueryResult<DialogmoteData>;
+  userType: "AG" | "SM";
```

Comment:
Vi bør forsøke å unngå prop-innsending, spesielt mtp å skille på AG og SM, og heller bruke komposisjon. Ellers vil vi ende opp med alt for mye spaghetti

Reply:
Fikset, fjernet bruken av SM og AG